autocorrelation.py

```diff
@@ -22,8 +22,34 @@
 import time
 
 DATABASE_TABLE = "atoms_b"
-DB_TICKER_QUERY = "data_json->>'ticker' = '{}' AND data_json->>'provider' = 'yahoo finance' ORDER BY data_json->>'datetime'"
-def query_lambda(ticker): return DB_TICKER_QUERY.format(ticker)
+
+
+def db_ticker_query(session, atoms_table, ticker):
+    """
```

Comment:
Also watch out for single quotes documentation comments for methods, I know it was probably like this from before

Reply:
Mb